CHANGES.md

```diff
@@ -1,3 +1,9 @@
+# vob 2.0.6 (2021-01-25)
+
+* Add `get_storage`.
```

Comment:
```suggestion
* Add `get_storage`, available when `unsafe_internals` is enabled.
```

Reply:
Good point. Fixed in https://github.com/softdevteam/vob/pull/55/commits/9a964ae74642c600b9a8c25bf1ddd4cea7d884a0. Probably best squashed before merging, assuming you're happy with my phrasing!